lizmap/server/server_info_handler.py

```diff
@@ -0,0 +1,96 @@
+__copyright__ = 'Copyright 2021, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+from typing import Union
+
+from qgis.core import Qgis
+from qgis.PyQt.QtCore import QRegularExpression
+from qgis.server import QgsServerOgcApi, QgsServerOgcApiHandler
+from qgis.utils import pluginMetadata, server_active_plugins
```

Comment:
@dmarteau Can you confirm that this list `server_active_plugins` will work on py-qgis-server ?
CF https://github.com/qgis/QGIS/blob/master/python/utils.py#L699

Reply:
Definitevely not because qgis server is embedded in python and python is not initialised from within c++ code).

Reply:
Fixed